Biz/Model/UserPasswordForm.pm

```diff
@@ -57,16 +58,20 @@ sub internal_pre_execute {
 }
 
 sub internal_validate_new {
+    my($self) = @_;
```

Comment:
I think this doesn't fix UserCreateForm, which also takes a password afaict.

My guess is it is better to increase the length of the Type.Password and add Type.PasswordLegacy, which is the type of password on the Login and "confirm" fields.